dists/go-ipfs/dist.json

```diff
@@ -2,5 +2,19 @@
   "id": "go-ipfs",
   "name": "go-ipfs",
   "tagline": "IPFS implementation and toolchain in Go",
-  "description": "go-ipfs is the main implementation of IPFS. It is the base distribution, and includes:\n- an IPFS core implementation\n- an IPFS daemon server\n- extensive command line tooling\n- an HTTP API for controlling the node\n- an HTTP Gateway for serving content to HTTP browsers\n"
-}
\ No newline at end of file
+  "description": "go-ipfs is the main implementation of IPFS. It is the base distribution, and includes:\n- an IPFS core implementation\n- an IPFS daemon server\n- extensive command line tooling\n- an HTTP API for controlling the node\n- an HTTP Gateway for serving content to HTTP browsers\n",
+  "platforms": {
+    "oses": [
+      {"id": "darwin",  "name": "Mac OSX App", "browser":"OS X",    "icon": "apple"},
+      {"id": "linux",   "name": "Linux App",   "browser":"Linux",   "icon": "linux"},
+      {"id": "openbsd", "name": "OpenBSD App", "browser":"OpenBSD", "icon": "circle-o"},
+      {"id": "freebsd", "name": "FreeBSD App", "browser":"FreeBSD", "icon": "circle-o"},
+      {"id": "windows", "name": "Windows App", "browser":"Windows", "icon": "windows"}
```

Comment:
I don't recall why i called this `<OS> App` -- go-ipfs is not an app, it's a cli toolchain.